### PR TITLE
Update BBC and BBC iPlayer icons

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
bbc

bbciplayer

**Issue:** No existing issue, but replaces earlier attempt to update these at #7751

**Similarweb rank:** 83

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  - Updates the BBC and BBC iPlayer logos to the new versions launched in October 2021.   
  - Updates brand colour for BBC iPlayer to that used on the iPlayer website.

### CODE REVIEW

File: icons/bbciplayer.svg
Comment (adamrusted) [inline icons/bbciplayer.svg:1]: I'm seeing a slight difference on this iPlayer logo from the SVG embedded in the header (overlaid in colour onto yours in black) happening in both diagonal arms of the logo...

image

Might be easily fixable by either re-rendering using that source, or upping the decimals when using SVGO.

File: icons/bbc.svg
Comment (adamrusted) [inline icons/bbc.svg:1]: There is an SVG baked into the BBC webpages which can be used.
Some minor differences compared to your icon - see website version in red.

image

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:1436]:
```suggestion
            "source": "https://www.bbc.co.uk",
```
SVG is available from the BBC website directly :smile:

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/bbciplayer.svg
Comment (paulrobertlloyd) [reply icons/bbciplayer.svg:1]: I can try updating these again, but was getting validation errors that I couldn’t get around without these adjustments. Will update again and see if those errors come back, and list them here.

Comment (adamrusted) [reply icons/bbciplayer.svg:1]: Yeah, if you run SVGO through [SVGOMG](https://jakearchibald.github.io/svgomg/) instead of the command line, you might need to set the precision decimals to 4. Sometimes these icons can just be like that 🙃

File: _data/simple-icons.json
Comment (paulrobertlloyd) [reply _data/simple-icons.json:1436]: Updated this and the SVG path in #f2de37863ed724a1be237d7337f60a8d82da722b, but not sure if that’s reflected in this PR now… I’m confused!

File: icons/bbciplayer.svg
Comment (paulrobertlloyd) [reply icons/bbciplayer.svg:1]: I get the following error using the SVG from the BBC website:

```
icon-centered <path> must be centered at (12, 12); the center is currently (12.002, 12)
```

I get this with a precision of 4 or 5; with 3 I get more errors. It’s hard to know how to fix this as I guess the centre is getting calculated based on where the slight curve of the point is being rendered.

Comment (adamrusted) [reply icons/bbciplayer.svg:1]: I'll take a quick look at the iPlayer one myself today, see if I get any issues. If not, I'll share my SVG path here so hopefully it'll work on your end too :)

File: icons/bbciplayer.svg
Comment (adamrusted) [inline icons/bbciplayer.svg:1]: Still seeing some slight differences between this PR (black) and the source file (red).
Feel free to use the below path - which to my eyes is a 100% match...
```
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BBC iPlayer</title><path d="M9.983.004a.142.142 0 0 0-.086.067L7.491 4.237a.141.141 0 0 0 .053.193l12.251 7.074a.141.141 0 0 0 .193-.053l2.405-4.166a.141.141 0 0 0-.053-.193L10.09.02a.142.142 0 0 0-.107-.016ZM1.731 4.785a.14.14 0 0 0-.1.042.141.141 0 0 0-.042.1v14.147c0 .039.017.074.043.1a.14.14 0 0 0 .1.04h4.81a.14.14 0 0 0 .14-.14V4.926a.14.14 0 0 0-.14-.14zm18.171 7.697a.141.141 0 0 0-.107.014L7.544 19.57a.142.142 0 0 0-.053.193l2.406 4.166c.02.034.051.057.086.067a.145.145 0 0 0 .107-.016l12.25-7.072a.142.142 0 0 0 .053-.193l-2.405-4.166a.142.142 0 0 0-.086-.067z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Couple of quick changes to make. If you use my path, we'll need another maintainer to approve changes.

File: _data/simple-icons.json
Comment (paulrobertlloyd) [reply _data/simple-icons.json]: Fixed in 6ae8dde

File: icons/bbciplayer.svg
Comment (paulrobertlloyd) [reply icons/bbciplayer.svg:1]: Fixed in 83ba471, and linting task now passing. Thank-you!

**Review by adamrusted — COMMENTED** [review]
LGTM! Though given my path was used, another member of @simple-icons/icon-review will need to confirm they're happy with it before we merge.

**Review by LitoMore — APPROVED** [review]

**Comment by adamrusted** [comment]
Merging as @LitoMore is happy with changes!

**Pull request merged**
